src/Cake.FileHelpers.Tests/Cake.FileHelpers.Tests.csproj

```
@@ -1,26 +1,23 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net50;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net461;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>

```

```suggestion
    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

`net461` is no longer a requirement, as of https://cakebuild.net/docs/extending/addins/best-practices#net-target-version

---

done